<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="mainGridLayout">
      <item row="21" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Layout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="24" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="btnSelectForceplateCycles">
        <property name="text">
         <string>Forceplate</string>
        </property>
       </widget>
      </item>
      <item row="22" column="1" colspan="2">
       <widget class="QCheckBox" name="xbKineticsFpOnly">
        <property name="text">
         <string>Plot kinetics for forceplate cycles only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="25" column="1">
       <widget class="QPushButton" name="btnPlot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gait cycles to plot:</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="23" column="1">
       <widget class="QCheckBox" name="xbLegend">
        <property name="text">
         <string>Include legend</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="21" column="2" colspan="3">
       <widget class="QComboBox" name="cbLayout">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gait cycles in selected trial:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="3" colspan="4">
       <widget class="NiceListWidget" name="listTrials">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Trials:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnAddC3DTrial">
        <property name="text">
         <string>Add C3D trials</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnAddNexusTrial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add Nexus trial</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="4">
       <widget class="QToolButton" name="btnPickCycles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QPushButton" name="btnSelect1stCycles">
        <property name="text">
         <string>1st R and L</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="btnSelectAllCycles">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" rowspan="3" colspan="4">
       <widget class="NiceListWidget" name="listTrialCycles"/>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="btnClearTrials">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="28" column="1">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="15" column="1" rowspan="3" colspan="4">
       <widget class="NiceListWidget" name="listCyclesToPlot"/>
      </item>
      <item row="19" column="1" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QPushButton" name="btnClearCyclesToPlot">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="25" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Average and plot</string>
        </property>
       </widget>
      </item>
      <item row="25" column="4">
       <widget class="QPushButton" name="btnSavePDF">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save PDF...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnAverageSelected">
        <property name="text">
         <string>Average selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionSave_as_PDF"/>
    <addaction name="actionNormal_data"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_as_PDF">
   <property name="text">
    <string>Save as PDF...</string>
   </property>
  </action>
  <action name="actionNormal_data">
   <property name="text">
    <string>Normal data...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NiceListWidget</class>
   <extends>QListWidget</extends>
   <header>plotter_gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
